Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py

```diff
@@ -1103,7 +1103,10 @@ def build_query(args, table_name):
     query_start_time, query_end_time = query_timestamp(args)
     timestamp_limitation = f'time_generated BETWEEN TIMESTAMP("{query_start_time}") AND ' \
                            f'TIMESTAMP("{query_end_time}") '
-    limit = args.get('limit', '5')
+    limit = args.get('limit')
+    # Added this to prevent an empty limit, if limit = '' the get will return '' and the query will fail
+    if not limit:
+        limit = '5'
```

Comment:
This also works and is more elegant
```suggestion
    limit = args.get('limit') or '5'
```